Training/NewTableViewCell.m

```diff
@@ -0,0 +1,19 @@
+//
+//  NewTableViewCell.m
+//  Training
+//
+//  Created by María Eugenia Sakuda on 3/4/15.
+//  Copyright (c) 2015 María Eugenia Sakuda. All rights reserved.
+//
+
+#import "NewTableViewCell.h"
+
+@implementation NewTableViewCell
+
+@synthesize favoriteImage = _favoriteImage;
```

Comment:
No hace falta esto. http://stackoverflow.com/questions/15606151/objective-c-is-synthesize-required-or-optional